webuni/commonmark-bundle/0.2/etc/packages/webuni_commonmark.yaml

```diff
@@ -0,0 +1,6 @@
+webuni_commonmark:
+    extensions:
+        # adds support for defining attributes on HTML elements (e.g. 'id', 'class', 'style')
+        attributes: true
+        # adds support for parsing Markdown tables
+        table: true
```

Comment:
Yes, but the problem is that the bundle doesn't have docs yet. This exposes the options so you can know about them: enabling the tables extension is a no-brainer to me, but the attributes extension is debatable.

About your comment, I thought the bundle installed both extensions ... but they are only in "require-dev". So, are you asking me to do changes in this reicpe or is your proposal something to be fixed in the bundle? Thanks!